src/index.js

```diff
@@ -1,79 +1,91 @@
-    document.querySelector(`#item${task.index}`).parentElement.classList.add('complete');
-  } else {
-    document.querySelector(`#item${task.index}`).parentElement.classList.remove('complete');
-    document.querySelector(`#item${task.index}`).checked = false;
-  }
-}
-const sorteddata = getDataLS();
-function diplayTask() {
-  if (sorteddata.length === 0) {
-    tasks.forEach((task) => {
-      addItemUI(task);
-      addTaskLS(task);
-    });
-  } else {
-    sorteddata.forEach((x) => {
-      addItemUI(x);
-    });
-  }
-}
-
+document.addEventListener('DOMContentLoaded', diplayTask);
 const list = document.querySelector('.list');
+document.querySelector('#enter').addEventListener('click', (e) => {
+  if (e.target.classList.contains('add-item')) {
+    const Input = document.querySelector('#add-item').value;
+    const index = getDataLS().length + 1;
+    const task = new Task(Input, index, false);
+    addTaskLS(task);
+    addItemUI(task);
+    clear();
+  }
+});
 list.addEventListener('change', (e) => {
   if (e.target.classList.contains('check')) {
     const task = document.querySelector(`#${e.target.id}`);
-    statueUdpdateLS(e);
+    const taskx = e.target;
+    statueUdpdateLS(taskx);
     statueUdpdateUI(task);
   }
 });
-document.addEventListener('DOMContentLoaded', diplayTask);
+list.addEventListener('click', (e) => {
+  if (e.target.classList.contains('delete')) {
+    remove(e.target.id);
+    e.target.parentElement.parentElement.remove();
+    window.location.reload();
+    indexValue();
+  }
+});
+document.querySelector('.clear-btn').addEventListener('click', () => {
+  let data = getDataLS();
+  data = data.filter((task) => !task.completed);
+  localStorage.setItem('tasks', JSON.stringify(data));
+  indexValue();
+  window.location.reload();
+});
+// list.addEventListener('click', (e) => {
+//   if (e.target.classList.contains('edit')) {
+//     const oldTask = e.target.parentElement.previousSibling.children[1];
+//     oldTask.classList.add('none');
+//     e.target.parentElement.previousSibling.children[2].classList.remove('none');
+//     getDataLS().forEach((el) => {
+//       if (el.description === oldTask.textContent) {
+//         localStorage.setItem('index', el.index);
+//       }
+//     });
+//     document.querySelector('.new').addEventListener('keypress', (e) => {
+//       if (e.key === 'Enter') {
+//         console.log(localStorage.getItem('index'));
+//       }
+//     });
+//   }
+// });
+// listItem.addEventListener('click', () => {
+//   const newInput = document.createElement('input');
+//   newInput.value = listItem.textContent;
+//   newInput.classList.add('editing')
+// });
+// const theNewInput = document.querySelector('.editing');
+// theNewInput.addEventListener('keypress', (e) => {
+//   if (e.key === 'Enter') {
+//     getDataLS().forEach((el) => {
+//       if (el.description === listItem.textContent) {
+//         console.log('adas');
+//       }
+//     })
+//   }
+// })
```

Comment:
- [ ] Let's remove those comments from your codebase since this makes it look messy.

Reply:
Thank you